### DISCUSSION TITLE
Add option to open modal in edit mode

### DISCUSSION THREAD

**Discussion by skamensky** [cortezaproject/corteza-server#784] [Ideas]
It would be very useful to have an option to open a modal in edit mode for edit heavy applications
![image](https://user-images.githubusercontent.com/19151369/212549523-8ebf2b43-bded-4a50-9097-03b7784ef3f8.png)

**Comment by Fajfa** [comment]
Would it make sense if the action icons decide in what mode to open the record.
And the dropdown where to open.

That means if you have it set up to open records in a modal and you click on the edit icon the record will be open in a modal in edit mode. Similar with other actions.

There could also exists a default mode the record should be opened in (view/edit/clone).

Thoughts?

**Reply by skamensky** [reply]
As an aside, for my use case I want edit to be the default mode everwhere the user has edit access.

In this case it be ideal if we could somehow control what mode (edit/view) the modal went into when they click anywhere on the row, not just the edit icon action (which is smaller and harder to find, especially if there are a lot of columns and you need to scroll to ther right to find the edit button).

It sounds like your second suggestion of a default mode would solve this for us. 

I'm not the strongest frontend dev but let me know if I can help in any way.
